src/views/DidEndpointsSign/DidEndpointsSign.tsx

```diff
@@ -16,14 +16,62 @@ import {
 } from '../../components/PasswordField/PasswordField';
 import { TxStatusModal } from '../../components/TxStatusModal/TxStatusModal';
 import { CopyValue } from '../../components/CopyValue/CopyValue';
-import { getKeystoreFromSeed } from '../../utilities/identities/identities';
+import {
+  getKeypairBySeed,
+  getKeystoreFromSeed,
+} from '../../utilities/identities/identities';
 import { useSubmitStates } from '../../utilities/useSubmitStates/useSubmitStates';
-import { getFragment, useFullDidDetails } from '../../utilities/did/did';
+import { getFragment, getFullDidDetails } from '../../utilities/did/did';
 import { generatePath, paths } from '../paths';
+import { useAsyncValue } from '../../utilities/useAsyncValue/useAsyncValue';
+import { KiltAmount } from '../../components/KiltAmount/KiltAmount';
+
+type ActionType = 'add' | 'remove';
+
+async function getDidAuthorizedExtrinsic(
+  seed: Uint8Array,
+  did: IDidDetails['did'],
+  endpoint: DidServiceEndpoint,
+  type: ActionType,
+) {
+  const keystore = await getKeystoreFromSeed(seed);
+  const keypair = getKeypairBySeed(seed);
+  const fullDidDetails = await getFullDidDetails(did);
+
+  // getRemoveEndpointExtrinsic expects just the fragment part, contrary to its type definition
```

Comment:
We can use this moment to drop the "contrary to its type definition" part, as the definition has been updated